schema/home/home_page_fairs_module.js

```diff
@@ -0,0 +1,35 @@
+import Fair from "schema/fair"
+import { GraphQLList, GraphQLObjectType } from "graphql"
+
+export const HomePageFairsModuleType = new GraphQLObjectType({
+  name: "HomePageFairsModule",
+  fields: {
+    results: {
+      type: new GraphQLList(Fair.type),
+      resolve: (root, options, request, { rootValue: { fairsLoader } }) => {
```

Comment:
The `options` parameter is not named the best in many of our resolvers alas, this parameters is actually the field’s arguments. Seeing as you haven’t actually defined any arguments for this field, this parameter should be unused and no need to merge it into the `gravityOptions` object below.